internal/filter/filterottl/filter_test.go

```diff
@@ -270,3 +271,45 @@ func Test_NewBoolExprForResource(t *testing.T) {
 		})
 	}
 }
+
+func Test_NewBoolExprForScope(t *testing.T) {
+	tests := []struct {
+		name           string
+		conditions     []string
+		expectedResult bool
+	}{
+		{
+			name: "basic",
+			conditions: []string{
+				"true == true",
+			},
+			expectedResult: true,
+		},
+		{
+			name: "multiple",
```

Comment:
Can you add a test for multiple lines resulting in false?

Reply:
Sure, I added this test.